internal/executor/rpc/rpc.go

```diff
@@ -227,6 +226,24 @@ func (r *RPC) ReportSingleCommand(
 	return &api.ReportSingleCommandResponse{}, nil
 }
 
+func (r *RPC) getCommandLogger(task *build.Task, commandName string) *echelon.Logger {
+	commandLoggerScope := fmt.Sprintf("'%s'", commandName)
+	// make pretty names for some instruction types
+	command := task.GetCommand(commandName)
+	switch {
```

Comment:
If we implement https://github.com/cirruslabs/cirrus-cli/pull/76/files#r489688469, we can simply use type assertion here.

Reply:
See 056773220f540420ab2b120da9101db961d1bae2